sql/load_demo_data.sql

```diff
@@ -6468,6 +6687,10 @@ CREATE TABLE `oh_visits` (
   PRIMARY KEY (`VST_ID`),
```

Comment:
The default value for the column VST_WRD_ID_A is NULL in the demo data and the empty string in the multiple steps.

Reply:
In this case, the field should be NOT NULL and it's also involved in a FK, so should remove the default value.
I think to address the change in OP-1107 with a new step_XX. I will report under the issue.
Thanks!